UBC-Meetable/code
# Tests, apidoc

## Objective
created some example jest tests to see how it would work with our codebase.
edited endpoints to work with the new flow and fixed bugs related to the flow
updated apidoc
manually tested flow to see that it works in at least some cases

## Testing instructions
for friend groups:
- create new user
- try updating the profile, make sure year and school are defined
- create quizInstance with full responses and uid of associated user and post it
- repeat previous steps for another user
- group one user, then the other, and check to see that all DB data is as expected.



## Checklist
- [ ] Linked Clubhouse Ticket
